test/topology_custom/test_tablets.py

```diff
@@ -22,6 +22,7 @@
 
 
 @pytest.mark.asyncio
+@pytest.mark.xfail(reason="issue #16129")
```

Comment:
Better test specifically the `ALTER KEYSPACE` below
`with pytest.raises(InvalidRequest, match="Changing replication options via ALTER KEYSPACE statement is not yet supported for keyspaces with tablets."):`

Reply:
for now...

Reply:
OK, asserting the error.